fs2/src/test/scala/laserdisc/fs2/RedisClientSpec.scala

```diff
@@ -65,47 +72,49 @@ final class RedisClientSpec extends WordSpecLike with Matchers with BeforeAndAft
         }).toList.sequence
 
       val testResponses =
-        (clientUnderTest[IO] evalMap {
-          cl => testPreset(cl) *> testRequests(cl)
+        (clientUnderTest[IO] evalMap { cl =>
+          testPreset(cl) *> testRequests(cl)
         }).compile.last map (
           _ getOrElse Nil
-        ) unsafeRunSync()
+        ) unsafeRunSync ()
 
-      testResponses.size should be (300)
-      testResponses map (_ should be (testPayload))
+      testResponses.size should be(300)
+      testResponses map (_ should be(testPayload))
     }
 
     "handle correctly some read requests in a row for a bulk text payload" in {
 
-      val testKey     = Key.unsafeFrom("test-key")
+      val testKey     = Key("test-key")
       val testPayload = (1 to 1000).toList map (_ => "test text") mkString " - "
 
       def testPreset[F[_]](cl: RedisClient[F])(implicit F: Monad[F]): F[Unit] =
         cl.send1(strings.set(testKey, testPayload)) flatMap (_ => F.unit)
 
       def testRequests[F[_]](cl: RedisClient[F])(implicit F: Concurrent[F]): F[List[String]] =
         ((1 to 50) map { _ =>
-           cl.send1(strings.get[String](testKey)) map (
-             _.fold(_ => "", _.getOrElse(""))
-           )
+          cl.send1(strings.get[String](testKey)) map (
+            _.fold(_ => "",
+                   _.getOrElse(""))
```

Comment:
is this scalafmt ? what do you think about 
```scala
_.fold(
  _ => "",
  _.getOrElse("")
)
```
for a multiline fold ?

Reply:
i believe so, scalafmt doesn't touch it